buddyboss/buddyboss-platform
# Custom Profile and Group tab support in Platform and Theme

### All Submissions:


Fixes #2489 .

### How to test the changes in this Pull Request:

1. Install [Modheader ](https://chrome.google.com/webstore/detail/modheader/idgpnmonknjnojddfkpgkljpfnnfcklj?hl=en-US)chrome extension and set header as shown: https://prnt.sc/15e9xa5
2. Make sure that buddyboss app plugin is active
3. visit any profile or group tab
4. only main content should be visible

### Proof Screenshots or Video
https://prnt.sc/15e9s97

### Other information:
